src/App.jsx

```diff
@@ -201,6 +201,11 @@ export class App extends React.Component {
       Toast(toastMessages.error ?? error.message, "error");
       return Promise.reject(error);
     })
+    // since we already display the erorr message. The caller does not need to catch the returned Promise.
```

Comment:
Should we just replace the `return Promise.reject(error)` above with `return false`?

I thought about doing something like this but wanted the error to be available to the caller for additional catching, but I think this is fine

Reply:
Should we just replace the return Promise.reject(error) above with return false?

I thought about this too, and almost did that exact thing.

But then I came to the same conclusion as you. I think it will be useful to have the error available to the caller.